src/components/AssetsDropZone/index.jsx

```diff
@@ -35,53 +35,60 @@ export default class AssetsDropZone extends React.Component {
-              { displayText => (
-                <Button
-                  className={['btn', 'btn-outline-primary']}
-                  label={displayText}
-                  onClick={this.handleClick}
-                  data-identifier="asset-drop-zone-browse-button"
-                />
-              )}
-            </WrappedMessage>
-          </div>
-          <WrappedMessage
-            message={messages.assetsDropZoneMaxFileSizeLabel}
-            values={{
-              maxFileSizeMB: this.props.maxFileSizeMB,
-            }}
-          >
-            { displayText => (
-              <p data-identifier="asset-drop-zone-max-file-size-label">
-                {displayText}
-              </p>
-            )}
-          </WrappedMessage>
-        </Dropzone>
-      </div>
+      <WrappedMessage message={messages.assetsDropZoneAriaLabel}>
+        { regionLabel =>
+          (
+            <div role="region" aria-label={regionLabel}>
+              <Dropzone
+                ref={this.setDropzonRef}
+                onDrop={this.onDrop}
+                className={styles['drop-zone']}
+                activeClassName={styles['drop-active']}
+                disableClick
+                maxSize={this.props.maxFileSizeMB * 1000000}
+                data-identifier="asset-drop-zone"
+              >
+                <p className="upload-icon" data-identifier="asset-drop-zone-icon">
+                  <span
+                    aria-hidden
+                    className={classNames(FontAwesomeStyles.fa, FontAwesomeStyles['fa-cloud-upload'])}
+                  />
+                </p>
+                <WrappedMessage message={messages.assetsDropZoneHeader}>
+                  { displayText => (
+                    <h2 data-identifier="asset-drop-zone-header">{displayText}</h2>
+                  )}
+                </WrappedMessage>
+                <div className={styles['center-text']}>
+                  <WrappedMessage message={messages.assetsDropZoneBrowseLabel} >
+                    { displayText => (
+                      <Button
+                        className={['btn', 'btn-outline-primary']}
+                        label={displayText}
+                        onClick={this.handleClick}
+                        data-identifier="asset-drop-zone-browse-button"
+                        aria-describedby="asset-drop-zone-max-file-size-label"
+                      />
+                    )}
+                  </WrappedMessage>
+                </div>
+                <WrappedMessage
+                  message={messages.assetsDropZoneMaxFileSizeLabel}
+                  values={{
+                    maxFileSizeMB: this.props.maxFileSizeMB,
+                  }}
+                >
+                  { displayText => (
+                    <p aria-hidden id="asset-drop-zone-max-file-size-label" data-identifier="asset-drop-zone-max-file-size-label">
+                      {displayText}
```

Comment:
Why are we hiding the max file size label from screenreaders? Don't they need to know it too?

Reply:
It's now a part of the aria-described by on the Browse button here: https://github.com/edx/studio-frontend/pull/179/files#diff-e5d111258070606d64e86dc6dbcd0a42R70

Reply:
Do screenreaders read out the contents of an element referenced by an `aria-describedby` even if that element has `aria-hidden`?

Reply:
Yeah - I tested on Safari with Voiceover and it reads out:

Browse your computer, button
Maximum file size: 10 MB, you are currently on a button, ....